pkg/convert/tf.go

```diff
@@ -1467,7 +1467,7 @@ func rewriteRelativeTraversal(scopes *scopes, fullyQualifiedPath string, travers
 		var name string
 		if fullyQualifiedPath != "" {
 			fullyQualifiedPath = appendPath(fullyQualifiedPath, attr.Name)
-			name = scopes.pulumiName(fullyQualifiedPath)
+			name = scopes.pulumiName(name, fullyQualifiedPath)
```

Comment:
name is "" here, it's only declared a couple of lines above

Reply:
Fixed.